packages/core/src/electron-main/electron-main-application.ts

```diff
@@ -268,10 +269,9 @@ export class ElectronMainApplication {
         };
     }
 
-    protected async openDefaultWindow(): Promise<BrowserWindow> {
-        const options = await this.getLastWindowOptions();
-        const [uri, electronWindow] = await Promise.all([this.createWindowUri(), this.createWindow(options)]);
-        electronWindow.loadURL(uri.toString(true));
+    async openDefaultWindow(): Promise<BrowserWindow> {
+        const [uri, electronWindow] = await Promise.all([this.createWindowUri(), this.createWindow()]);
+        electronWindow.loadURL(uri.withFragment(DEFAULT_WINDOW_HASH).toString(true));
```

Comment:
This change means that we will not restore the previous workspace if we do the following:
1. start the `example-electron` application
2. open a workspace folder
3. close the application
4. re-start the application - the previous workspace is not restored and instead we fallback to the default state (without a workspace).